DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/ViewDelegate/ViewDelegateViewModel.cs

```diff
@@ -20,10 +20,15 @@ IEnumerable<DelegateCourseDetails> delegateCourses
             DelegateInfo = new DelegateInfoViewModel(delegateUser, customFields);
             DelegateCourses = delegateCourses.Select(x => new DelegateCourseInfoViewModel(x)).ToList();
             Tags = FilterableTagHelper.GetCurrentTagsForDelegateUser(delegateUser);
+
+            ProfessionalRegistrationNumber = delegateUser.HasBeenPromptedForPrn
+                ? delegateUser.ProfessionalRegistrationNumber ?? "Not professionally registered"
+                : "Not yet provided";

```

Comment:
Is there a helper method or a delegateUser property that sets this that could be used on the My Account page as well? Or do they not share enough to be commonised?

Reply:
Good idea, added a method on DelegateUser for it